about/faq.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "WPE's Frequently Asked Questions"
+tags: [about]
+data: { dateless: "true" }
+permalink: /about/faq.html
+---
+
+[[toc]]
+
+## What is the difference between WebPlatformForEmbedded's WPE and WPEWebKit?
+
+Historically the [WebPlatformForEmbedded flavor of
+WPE](https://github.com/WebPlatformForEmbedded/WPEWebKit) came first. This
+version includes several adaptations for the RDK platform, as well as different
+fixes for its supported devices. The [RDK WPE
+page](https://wiki.rdkcentral.com/display/RDK/WPE) provides more information
+about WPE in the RDK platform.
+
+In 2017 the Igalia folks submitted a new flavor of WPE, suitable for upstream
+hosting under the webkit.org umbrella. This version of WPE its released every 6
+months from the code hosted on the [upstream repository](https://webkit.org).
+This flavor of WPE is maintained upstream and provides regular security updates.
+
+## What Web features does WPE support?
+
+The WebKit project provides a [features status page](https://webkit.org/status/)
```

Comment:
These are things supported by WebKit, but it doesn't necessarily mean they are supported in all ports, right?  Won't some also depend on the backend you choose, at least?

Reply:
Right, this needs more details as flagged by the TODO.